### DISCUSSION TITLE
How to change the border color of the selected table row at TR level

### DISCUSSION THREAD

**Discussion by 549393092** [quasarframework/quasar#14885] [General - Components / Directives / etc]
Dear Quasar team,

I would like to ask if it's possbile to set the border color of the selected table row at TR level.
I did a demo to explain it, could you please have a look?
https://codepen.io/Stanley-549393092/pen/OJEmYbW?editors=1011
As you can see, it can be done at TD (cell) level, but it has to add it to all the column definition.
I tried a lot ways to to set it at TR (row) level, but it does not work.

Regards

**Comment by 549393092** [comment]
There is a thread about this topic.
https://stackoverflow.com/questions/18679020/border-around-tr-element-doesnt-show
I tried to change table border-collapse from separated to collapse, but it still not work.
Finally, I found an workaround of using box-shadow.
see the demo. https://codepen.io/Stanley-549393092/pen/poKwNbp?editors=1011
But the solution is still not perfect, as it cannot show shadow for the last row. Hope someone can figure it out.

Regards

**Comment by 549393092** [comment]
Finally it's solved with below demo.
https://codepen.io/Stanley-549393092/pen/yLEoNXp?editors=1111
Still, not perfect, I would like to set the width as 1px, but it will not change the color.
```
border-bottom: 2px solid #1976D2 // change to 1px will not work
```
I tried to set it as 1.1px, it works in edge, but not work in firefox.
Could someone have a look, thanks!

**Reply by pdanpdan** [reply]
https://codepen.io/pdanpdan/pen/WNyEGwq?editors=1111

**Reply by 549393092** [reply]
@pdanpdan Thanks for your help! But it seems it not works for the last row.
